great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py

```diff
@@ -1,8 +1,9 @@
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, cast
 
 from great_expectations.core.batch import (
     BatchDefinition,
     BatchRequest,
+    BatchRequestBase,
```

Comment:
Why are we importing `BatchRequestBase`?  Is it being used?  Sorry if GitHub obscures the full view.

Reply:
youre right :) this shouldn't be here anymore